src/scripts/main.js

```diff
@@ -1,5 +1,49 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+
 document.addEventListener('click', e => {
-  // write code here
+  const wall = e.target.closest('.wall');
+
+  if (!wall) {
+    return;
+  }
+
+  const clickCoordinateX = e.clientX;
+  const clickCoordinateY = e.clientY;
+  const wallCoordinates = wall.getBoundingClientRect();
+  const wallCoordinateX = wallCoordinates.x;
+  const wallCoordinateY = wallCoordinates.y;
+  const wallWidth = wall.clientWidth;
+  const wallHeight = wall.clientHeight;
+  const wallBorderLeft = wall.clientLeft;
+  const wallBorderTop = wall.clientTop;
+
+  const spiderWidth = spider.clientWidth;
+  const spiderHeight = spider.clientHeight;
+
+  let calculatedSpiderLeft = (clickCoordinateX - wallCoordinateX
+    - wallBorderLeft - spiderWidth / 2);
```

Comment:
```suggestion
  let calculatedSpiderLeft = (
     clickCoordinateX - wallCoordinateX - wallBorderLeft - spiderWidth / 2
  );
```
я думаю должно поместиться